packages/storage/README.md

```diff
@@ -482,6 +482,128 @@ if (currentPage.data) {
 console.log(allFiles);
 ```
 
+## Creating a bucket
+
+`createBucket` function can be used to create a new bucket.
+
+### `createBucket`
+
+```ts
+createBucket(bucketName: string, options?: CreateBucketOptions): Promise<TigrisStorageResponse<CreateBucketResponse, Error>>;
+```
+
+`createBucket` accepts the following parameters:
+
+- `bucketName`: (Required) A string specifying the name of the bucket to create
+- `options`: (Optional) A JSON object with the following optional parameters:
+
+#### `options`
+
+| **Parameter**        | **Required** | **Values**                                                                       |
+| -------------------- | ------------ | -------------------------------------------------------------------------------- |
+| enableSnapshot       | No           | Enable snapshot versioning for the bucket. Default is `false`.                   |
+| sourceBucketName     | No           | The name of the source bucket to fork from.                                      |
+| sourceBucketSnapshot | No           | The snapshot version of the source bucket to fork from.                          |
+| config               | No           | A configuration object to override the [default configuration](#authentication). |
+
+In case of successful `createBucket`, the `data` property will be set and contains the following properties:
+
+- `isSnapshotEnabled`: Whether snapshot versioning is enabled for the bucket
+- `hasForks`: Whether the bucket has forks
+- `sourceBucketName`: The name of the source bucket (if created from a fork)
+- `sourceBucketSnapshot`: The snapshot version of the source bucket (if created from a fork)
+
+### Examples
+
+#### Create a simple bucket
+
+```ts
+const result = await createBucket('my-new-bucket');
+
+if (result.error) {
+  console.error('Error creating bucket:', result.error);
+} else {
+  console.log('Bucket created successfully:', result.data);
+}
+```
+
+#### Create a bucket with snapshot versioning enabled
+
+```ts
+const result = await createBucket('my-versioned-bucket', {
```

Comment:
```suggestion
const result = await createBucket('my-snapshot-bucket', {
```

Reply:
Versioned buckets are coming soon. Might be better to not use such naming right now to avoid confusion.